### ISSUE TITLE
The method     'UnmodifiableUint8ListView' isn't defined for the class 'CroppedFile'.

### ISSUE THREAD

**Issue by donywan** [serverpod/serverpod#2686]
**Describe the bug**
```
/d:/pub_cache/hosted/pub.flutter-io.cn/image_cropper_platform_interface-5.0.0/lib/src/models/cropped_file/html.dart:32:38: Error: The method    
'UnmodifiableUint8ListView' isn't defined for the class 'CroppedFile'.
 - 'CroppedFile' is from 'package:image_cropper_platform_interface/src/models/cropped_file/html.dart'
 ('/d:/pub_cache/hosted/pub.flutter-io.cn/image_cropper_platform_interface-5.0.0/lib/src/models/cropped_file/html.dart').
Try correcting the name to the name of an existing method, or defining a method named 'UnmodifiableUint8ListView'.
      return Future<Uint8List>.value(UnmodifiableUint8ListView(_initBytes!));
```

**To Reproduce**
1. `cd projectname_server` and `dart pub add serverpod_auth_server`
2. start `projectname_server`
3. add package `serverpod_auth_shared_flutter` to `projectname_flutter`
4. start `projectname_flutter`

**Expected behavior**
`projectname_flutter` start work.

**Screenshots**

**Serverpod versions**
 -  _CLI Version:_ Serverpod version: 2.0.2
 - _Library version:_ 2.0.2

**Platform information**
`EDGE `

**Additional context**

**Comment by Isakdl** [comment]
This seems to be an issue in the package: image_cropper_platform_interface and shouldn't be related to the serverpod code.

**Comment by donywan** [comment]
@Isakdl Yes, `serverpod_auth_shared_flutter` depends on `image_cropper: ^5.0.0`, and `image_cropper` depends on `image_cropper_platform_interface: ^5.0.0`.

**Comment by donywan** [comment]
@Isakdl The `UnmodifiableUint8ListView` method has been removed in `dart 3.5.1`, so you need to upgrade your dependencies.

**Comment by Isakdl** [comment]
Ah got it, this is solved on main already and is fixed in the upcoming 2.1 release, beta available on pub.dev
